app/js/lib/foxbox/recipes.js

```diff
@@ -28,8 +28,9 @@ export default class Recipe {
       `${this[p.net].origin}/api/v${this[p.settings].apiVersion}/services`)
       // Get all recipes.
       .then(services => services.filter(service =>
-        service.adapter === 'thinkerbell-adapter' &&
-        service.id !== 'thinkerbell-root-service'
+        service.adapter != 'thinkerbell-adapter'
+        && service.adapter != 'webpush@link.mozilla.org'
```

Comment:
__nit:__ when boolean condition is wrapped it's usually indented to keep to make it easier to track single condition, like:
```
service.adapter != 'thinkerbell-adapter'
  && service.adapter != 'webpush@link.mozilla.org'
  && service.adapter != 'console@link.mozilla.org'
```

And we need to agree on ```!==``` or ```!=```, triple sign feels safer in JS world, but I'm fine with any choice :)